examples/ding-dong-bot.ts

```diff
@@ -34,13 +34,13 @@ import { generate } from 'qrcode-terminal'
  */
 const bot = new Wechaty({
   name : 'ding-dong-bot',
-  
+
```

Comment:
Please remove unrelated changes: keep the blank link between those comments to make them separate clearer.

Reply:
Thanks Huan, I undestand. But following lint issue stopped me while pushing:

/Users/.../github/wechaty/examples/ding-dong-bot.ts
  37:1  error  Trailing spaces not allowed  no-trailing-spaces
  43:1  error  Trailing spaces not allowed  no-trailing-spaces

And there are actually no trail spaces on other empty lines in this file, so I updated code to fix the lint issue.
What's your suggestion for next step then?

Reply:
That's OK, let's merge this update first.

Thank you very much!